ludwig/schema/trainer.py

```diff
@@ -226,12 +307,6 @@ class GBMTrainerConfig(BaseTrainerConfig):
         allow_none=False,
     )
 
-    eval_batch_size: int = schema_utils.PositiveInteger(
```

Comment:
Just want to double check, `eval_batch_size` here had the `default` set to `128` whereas the `default` on `BaseTrainerConfig` is `None`